README.md

```diff
@@ -46,21 +58,23 @@ seneca.ready(function(){
 ```
 
 ## Usage
-You don't use this module directly. It provides an underlying data storage engine for the Seneca
-entity API:
+
+You don't use this module directly. It provides an underlying data storage engine for the Seneca entity API:
 
 ```js
 var entity = seneca.make$('typename')
 entity.someproperty = "something"
 entity.anotherproperty = 100
 
-entity.save$(function (err, entity) { ... })
-entity.load$({id: ...}, function (err, entity) { ... })
-entity.list$({property: ...}, function (err, entity) { ... })
-entity.remove$({id: ...}, function (err, entity) { ... })
+entity.save$( function (err,entity) { ... } )
+entity.load$( {id: ...}, function (err,entity) { ... } )
+entity.list$( {property: ...}, function (err,entity) { ... } )
+entity.remove$( {id: ...}, function (err,entity) { ... } )
 ```
```

Comment:
remove padding on each end and add a space after commas eg `entity.save$(function (err, entity) { ... })`